docs/activities/tf-td/index.html

```diff
@@ -4,8 +4,59 @@
 group: "navigation"
 ---
 <div>
-	<h2>W3C WoT – ThingDescription Task Force</h2>
-	<p>
-		Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
-	</p>
-</div>
+	<h2>W3C WoT – Thing Description Task Force</h2>
+		<p>The WoT Thing Description task force is responsible for 
+		defining the information model for the Things, its interpretation and
+		common representation. In addition, the task force covers WoT Binding Templates 
+		topics to define the mapping to concrete IoT protocols and payload encodings.		
+		</p>
+		<div>
+		<h3>Normative Deliverables</h3>
+			<p>Current work is focused on a normative deliverables (WoT Thing Description 1.1) and on a informative deliverables (WoT Binding Templates).
+			</p>
+			<div>
```

Comment:
I suggest to have a `<h3>` for normative deliverables were one can find the TD document and afterwards another `<h3>` for non-normative deliverables were we put bindings templates.

Reply:
done